mcs/class/corlib/System.Threading/Thread.cs

```diff
@@ -53,7 +53,7 @@ sealed class InternalThread : CriticalFinalizerObject {
 		IntPtr native_handle; // used only on Win32
 		IntPtr unused3;
 		/* accessed only from unmanaged code */
-		private IntPtr name;
+		private byte* name_utf8;
```

Comment:
This requires corlib version bump

Reply:
I thought so, thank you, not this part, but other. I asked Ludovic & he confirmed.